modules/context/repo.go

```diff
@@ -1081,6 +1094,9 @@ func (ctx *Context) IssueTemplatesErrorsFromDefaultBranch() ([]*api.IssueTemplat
 			return issueTemplates, nil
 		}
 		for _, entry := range entries {
+			if entry.Name() == "config.yaml" || entry.Name() == "config.yml" {
```

Comment:
The check isn't necessary, `template.CouldBe` will return false if it's config file.